.rubocop.yml

```diff
@@ -19,4 +19,7 @@ RSpec/FilePath:
   SpecSuffixOnly: true
 
```

Comment:
What is this file? I'm very curious.

Reply:
I've installed a gem called [rubocop](https://docs.rubocop.org/rubocop/index.html) - it's a linter that I use to determine if my code is in alignment with ruby/rails style guide rules. I'm using this file specifically to disable some [rules/cops](https://docs.rubocop.org/rubocop/cops_style.html) and prevent rubocop from alerting me about my code's non-compliance 😆 (some of those rules are a bit too annoying to comply with)

Reply:
I hear that. I love those tools, but sometimes they can be quite oppressive with their pickiness. I'm going to check it out!